qualtran/bloqs/mod_arithmetic/mod_multiplication.py

```diff
@@ -265,3 +273,463 @@ def _modmul_symb() -> CModMulK:
 
 
 _C_MOD_MUL_K_DOC = BloqDocSpec(bloq_cls=CModMulK, examples=(_modmul_symb, _modmul))
+
+
+@frozen
+class SingleWindowModMul(Bloq):
+    r"""Performs modular multiplication on a single windowed.
+
```

Comment:
"used as a subroutine in DirtyModMul"

Reply:
done

Reply:
Sorry, I meant to add this as an additional line instead of replacing the original. The original description should probably be kept if this bloq can indeed be used standalone

Reply:
done